packages/maskbook/src/_locales/zh/messages.json

```diff
@@ -1,682 +1,682 @@
 {
     "about": "關於",
-    "about_dialog_description": "Mask Network 允許您在社交網絡上發布加密的帖子，從而可以減輕大公司對大數據的監視。只有您和您的朋友（也在使用 Mask Network）可以解密您的加密帖子。",
-    "about_dialog_touch": "保持聯繫",
-    "about_dialog_feedback": "提供反饋意見：",
-    "about_dialog_source_code": "程式源代碼：",
-    "about_dialog_license": "許可証：",
-    "add_to_key_store_success": "Mask 已獲取此公鑰並添加至本地",
-    "add_token": "添加代幣",
-    "add_token_search_hint": "搜索 ERC20 代幣",
-    "add_asset": "",
-    "additional_post_box__encrypted_post_pre": "來 #mask_io 解密這個訊息！ {{encrypted}}",
-    "additional_post_box__steganography_post_pre": "此圖片僅分享給我的好友。請安裝 mask.io 以解讀圖片。 ([I:b]) {{random}}",
-    "auto_paste_failed_dialog_title": "手動貼入加密内容",
-    "auto_paste_failed_dialog_content": "請右鍵複製以下圖片（若有）和文字並發佈到社交網絡上。",
-    "auto_paste_failed_dialog_image_caption": "在新窗口中打開",
-    "auto_paste_failed_snackbar": "您需要手動貼入 Mask 加密后的内容嗎？",
-    "auto_paste_failed_snackbar_action": "開始吧",
-    "automation_request_click_post_button": "請點擊「發布」按鈕以打開編輯對話框。",
-    "new_avatar_url": "新的頭像網址",
-    "new_avatar_url_placeholder": "無需修改時請留空",
+    "about_dialog_description": "Mask Network是通向新的、開放的Internet的門戶。通過Mask Network，您可以向朋友發送加密貼文，參與加密貨幣抽獎，並在您已經使用的平台上共享加密文件。",
+    "about_dialog_touch": "聯絡我們",
+    "about_dialog_feedback": "提供建議: ",
+    "about_dialog_source_code": "原始碼: ",
+    "about_dialog_license": "授權: ",
+    "add_to_key_store_success": "公開金鑰已添加到金鑰庫",
+    "add_token": "新增代幣",
+    "add_token_search_hint": "搜尋代幣",
```

Comment:
`ERC20` modification, I think it can't be removed.

Reply:
fixed via e2d988da